subjects/countstars/README.md

```diff
@@ -0,0 +1,33 @@
+## CountStars
+
+### Instructions
+Write a function named `CountStar` that makes you fall asleep by counting stars, it takes an integer in the parameter
```

Comment:
Try this instead:

`Write a function called CountStars that counts the stars up to a number given as an argument.`